# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/external-memory/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/external-memory/.cargo-checksum.json b/third_party/rust/external-memory/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/external-memory/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"3b75de4bf665b60c9fda3f35ca306b056a4717a528b64a857e37fe4905e6cd78","src/fd.rs":"ec0ef055ae91ce42ad48667b09fadffc06ffb24ab79330be62a932d7a501d1ae","src/handle.rs":"f8c0abafee759e6802c73b2366ea0fbabd315bf9258b2399bf20542a5773baca","src/lib.rs":"dc18d38876f40868d3c624af260276b88e42113cdf103a4fbcd47ba1e29ea2e5","src/ptr.rs":"c6492930679f24e4f5fd9ac9221183fdd0f35ecfb1e78ea09ec65c5cd6083241"},"package":"e4dfe8d292b014422776a8c516862d2bff8a81b223a4461dfdc45f3862dc9d39"}
\ No newline at end of file